src/Customizer.php

```diff
@@ -75,6 +76,10 @@ public function register() {
 			'default'           => $settings_defaults['sticky_selected'],
 			'sanitize_callback' => 'sanitize_key',
 		) );
+		$this->wp_customize->add_setting( 'sticky_menu_layout', array(
```

Comment:
@krumpak the setting name is still "layout", this should be changed to "visibility"